src/main/webapp/css/loginpage.css

```diff
@@ -0,0 +1,83 @@
+
+
+body {
+  text-align:center;
+  margin: 0;
+  font-family: sans-serif;
+  line-height: 1.6;
+  font-size: 18px;
+  line-height: 1.6;
+  color: #444;
+  background-color: #eeeeee;
+}
+
+.navbar-brand {
+  padding: 0 15px;
+  height: 200px;
+  line-height: 80px;
+}
+
+nav {
+  background-color: lightblue;
+  line-height: 3;
+}
+
+
+nav b {
+  text-align: center;
+  background-color: darkslategrey;
+  line-height: 8;
+  column-width: 100000px;
+  color: white;
+  display: inline-block;
+  font-size: 24px;
+  text-decoration: none;
+  font-family: monospace;
+  font-weight: bolder;
+}
+
+#navTitle {
+  font-size: 36px;
+}
+
+#container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 800px;
+}
+
+h1 {
```

Comment:
This means that all h1 in the site will have the specified color. If that is intended for the whole site instead of just this page, consider creating a "global.css" that applies to all pages so they can look consistent and the code can be reused. If this color is specified to the h1 in this page, it might be clearer to use a class or id since there may eventually be multiple h1 in this page.